database/clickhouse/clickhouse.go

```diff
@@ -192,7 +201,7 @@ func (ch *ClickHouse) SetVersion(version int, dirty bool) error {
 		return err
 	}
 
-	query := "INSERT INTO " + ch.config.MigrationsTable + " (version, dirty, sequence) VALUES (?, ?, ?)"
+	query := "INSERT INTO " + ch.config.DatabaseName + "." + ch.config.MigrationsTable + " (version, dirty, sequence) SETTINGS distributed_foreground_insert = 1 VALUES (?, ?, ?)"
```

Comment:
shouldn't the setting be for `x-distributed` only?

Reply:
Ah yeah it's probably cleaner to do so. Though I tested it and it wouldn't have actually broken things 

```
clickhouse02 :) show create table schema_migrations

SHOW CREATE TABLE schema_migrations

Query id: 6b6535ff-b732-4982-b6f3-671d79af002d

┌─statement───────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ CREATE TABLE default.schema_migrations
(
    `version` Int64,
    `dirty` UInt8,
    `sequence` UInt64
)
ENGINE = TinyLog │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

1 row in set. Elapsed: 0.001 sec. 

clickhouse02 :) insert into schema_migrations (version, dirty, sequence) settings  distributed_foreground_insert = 1 values (1, 2 ,3)

INSERT INTO schema_migrations (version, dirty, sequence)
SETTINGS distributed_foreground_insert = 1
FORMAT Values

Query id: 766aaaa1-6453-4244-865a-9613e0d6c994

Ok.

1 row in set. Elapsed: 0.021 sec
```